assets/agenda/components/AgendaListItem.tsx

```diff
@@ -185,17 +194,27 @@ class AgendaListItem extends React.Component<any, any> {
 
                         {(isMobile || isExtended) && description && (
                             <div className="wire-articles__item__text">
-                                {item.es_highlight && item.es_highlight
+                                {item.es_highlight && item.es_highlight.definition_short
                                     ? (
-                                        description.split('\n').map((lineOfHTML: string, index: number) => {
-                                            return (
-                                                <p key={index}>
-                                                    <span dangerouslySetInnerHTML={{__html: lineOfHTML}} />
-                                                </p>
-                                            );
+                                        descriptionArray.split('\n').map((lineOfHTML: string, index: number) => (
+                                            <p key={index}>
+                                                <span dangerouslySetInnerHTML={{__html: lineOfHTML}} />
+                                            </p>
+                                        ))
+                                    )
+                                    : (
+                                        descriptionArray.split('\n').slice(0, 3).map((plainOfHTML: string, i: number, array: any) => {
+                                            const lastChild: boolean = array.length -1 === i && descriptionArray.length > 3;
```

Comment:
wouldn't it be easier to use `:last-child` in css?

Reply:
not really... for example if you only have two rows of description then you don't need to show three dot on the second one (lastchild in this case)...

Reply:
I see, makes sense then